tests/rptest/test_many_partitions.yml

```diff
@@ -0,0 +1,9 @@
+# TODO work in progress getting clustered mode issues worked out
+partitions:
+  included:
+    - scale_tests/many_partitions_test.py
+
+  # excluded:
+  #   - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_many_partitions
+  #   - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_many_partitions_compacted
+  #   - scale_tests/many_partitions_test.py::ManyPartitionsTest.test_omb
```

Comment:
Perhaps these _should_ be excluded? They're otherwise running in CI just fine.

Reply:
Yes, they should be excluded, I guess. Just wanted them to to go along to see other tests passing next to the one failing before it will be stabilized.